tests/specs/bihu/verification.k

```diff
@@ -25,17 +33,14 @@ module VERIFICATION
        andBool  I2 <=Int I3
       [simplification]
 
-    rule 0 <=Int X /Int Y             => true requires 0 <=Int X andBool X <Int pow256 andBool 0 <Int  Y andBool Y <Int pow256
-      [simplification]
-    rule         X /Int Y <Int pow256 => true requires 0 <=Int X andBool X <Int pow256 andBool 0 <Int  Y andBool Y <Int pow256
-      [simplification]
+    rule 0 <=Int X /Int Y             => true requires 0 <=Int X andBool X <Int pow256 andBool 0 <Int  Y andBool Y <Int pow256 [simplification]
+    rule         X /Int Y <Int pow256 => true requires 0 <=Int X andBool X <Int pow256 andBool 0 <Int  Y andBool Y <Int pow256 [simplification]
 
     rule I1 +Int I2 <Int I1 => false requires 0 <=Int I2 [simplification]
 
     rule I1 <Int I1 -Int I2 => false requires 0 <=Int I2 [simplification]
 
-    rule chop(I1 -Int I2) => I1 -Int I2 requires 0 <=Int I1 andBool I2 <Int pow256 andBool I1 >=Int I2
-      [simplification]
+    rule chop(I1 -Int I2) => I1 -Int I2 requires 0 <=Int I1 andBool I2 <Int pow256 andBool I1 >=Int I2 [simplification]
```

Comment:
Is this lemma needed? It looks untrue to me. Take for example, `I1 ==Int 10`, `I2 ==Int 0 -Int pow256`, then we get that `I1 -Int I2 => 10 +Int pow256`, so that `chop(I1 -Int I2) =/=Int I1 -Int I2`.